packages/material-ui/src/FormHelperText/FormHelperText.js

```diff
@@ -64,21 +88,25 @@ const FormHelperText = React.forwardRef(function FormHelperText(props, ref) {
     states: ['variant', 'size', 'disabled', 'error', 'filled', 'focused', 'required'],
   });
 
+  const styleProps = {
+    ...props,
+    contained: fcs.variant === 'filled' || fcs.variant === 'outlined',
+    variant: fcs.variant,
+    small: fcs.size === 'small',
```

Comment:
```suggestion
    size: fcs.size,
```

For consistency with other components,let's use the `size`.